test/fixtures/impersonation-requests/impersonationRequests.ts

```
@@ -0,0 +1,122 @@
+import { REQUEST_STATE } from "../../../constants/requests";
+
+export const createImpersonationRequestBody = {
+  impersonatedUserId: "user123",
+  reason: "Debugging user issue"
```

ESLint

[error] Insert `,`

(prettier/prettier)